src/compositions/InputGroup/utils/schemas.ts

```diff
@@ -0,0 +1,67 @@
+import * as yup from "yup";
+
+export const Text = yup.string();
+
+export const Email = yup.string()
+  .email('The field value must be a valid email address');
+
+export const Subject = yup.string()
```

Comment:
Subject is more complex than that:
"Subject: string
pattern: ^[A-Za-z0-9_]{1,64}$"